scripts/run_node.sh

```diff
@@ -160,8 +165,14 @@ then
     if [ -z "$ARCHIVIST" ]
     then
         ###### VALIDATOR ######
-        source env/validator
-        eval "echo \"$(cat env/validator)\"" > env/validator.env
+        if [ -n "$MAINNET" ]
+        then
+            source env/validator_mainnet
+            eval "echo \"$(cat env/validator_mainnet)\"" > env/validator.env
```

Comment:
Doing this that way will override env for testnet, right? Probably, we should change testnet env name for `validator_testnet.env`.

Reply:
I can change that :)

Reply:
Seems like not necessary as it's ephemeral and regenerated on every start. Resolving.